server/api/internal/infrastructure/websocket/client.go

```
@@ -50,85 +45,90 @@
-
-	log.Infof("Calling FFI get_latest_document with docID: %s", docID)
-	result := C.get_latest_document(cDocID, cConfigJSON)
-	if result == nil {
-		log.Errorf("FFI call returned nil for docID: %s", docID)
-		return nil, fmt.Errorf("failed to get latest document")
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to send request: %w", err)
 	}
-	defer C.free_string(result)
+	defer resp.Body.Close()
```

golangci-lint (1.62.2)

Error return value of `resp.Body.Close` is not checked

(errcheck)